instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/config/ExperimentalConfig.java

```diff
@@ -38,6 +38,11 @@ public boolean viewTelemetryEnabled() {
         "otel.instrumentation.common.experimental.view-telemetry.enabled", !suppressViewSpans);
   }
 
+  public boolean useViewNameAsHttpRoute() {
+    return config.getBoolean(
+        "otel.instrumentation.common.experimental.view-name-as-http-route.enabled", false);
+  }
```

Comment:
do you have a need for this? if not, I'm always in favor of only adding config options on demand 😄

Reply:
No, I don't, I added it "just to be on the safe side" 😂 

Let's drop it 👍